### ISSUE TITLE
Audio to phoneme?

### ISSUE THREAD

**Issue by jiunn95** [rand0musername/speech-rec#1]
Hi is this project about inputting wav file / audio file and it outputs the existing phoneme?
If so, is there a documentation?

**Comment by rand0musername** [comment]
Yes. I never documented this properly, but `lstm_main.py` is the entry point and the dataset should be set up similar to `timit_dataset.py`.
